test/Dapr.Actors.Test/ActorMethodInvocationExceptionTests.cs

```diff
@@ -25,21 +25,25 @@ public class ActorMethodInvocationExceptionTests
         public void TestThrowActorMethodInvocationException()
         {
             // Create Remote Actor Method test Exception
-            var exception = new InvalidOperationException();
-            var message = "Remote Actor Exception";
+            var message = "Remote Actor encountered an exception";
+            var innerMessage = "Bad time zone";
+            var exception = new InvalidOperationException(message, new InvalidTimeZoneException(innerMessage));
 
             // Create Serialized Exception
-            (var serializedException, _) = RemoteException.FromException(new InvalidOperationException());
+            (var serializedException, _) = ActorCommunicationException.FromException(exception);
 
             // De Serialize Exception
-            var isDeserialzied = RemoteException.ToException(
+            var isDeserialzied = ActorCommunicationException.ToException(
                                                      new MemoryStream(serializedException),
                                                      out var remoteMethodException);
             isDeserialzied.Should().BeTrue();
             var ex = this.ThrowRemoteException(message, remoteMethodException);
             ex.Should().BeOfType<ActorMethodInvocationException>();
-            ex.InnerException.Should().BeOfType<InvalidOperationException>();
+            ex.InnerException.Should().BeOfType<ActorCommunicationException>();
+            ((ActorCommunicationException)ex.InnerException).ActualExceptionType.Should().Be("System.InvalidOperationException");
+            ex.InnerException.InnerException.Should().BeNull();
             ex.Message.Should().Be(message);
+            ex.InnerException.Message.Should().Be(message);
```

Comment:
We should probably start adding integration tests for Actors - it might be a good thing to follow up on after this PR (which is already large).

Reply:
Marked this for milestone 3: https://github.com/dapr/dotnet-sdk/issues/389